plugin/al/lib/AL_USDMaya/AL/usdmaya/fileio/TransformIterator.h

```diff
@@ -128,14 +128,15 @@ class TransformIterator
 
   std::vector<StackRef> m_primStack;
   UsdStageRefPtr m_stage;
+  size_t m_currentItem;
```

Comment:
/git/usd/maya-usd/plugin/al/lib/AL_USDMaya/AL/usdmaya/fileio/TransformIterator.h:131:10: error: private field 'm_currentItem' is not used [-Werror,-Wunused-private-field]
  size_t m_currentItem;

As well, for variables like this (and m_stopOnInstance) below, use in-class member initializers instead (https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rc-default)